src/Pages/kiwanlee/Main/Main.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+
+import './Main.scss';
+import Nav from '../../../Components/Nav/Nav';
+//클래스형 표현
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      noDisplaying: 'none',
+      displaying: 'inline-block',
+      comment: '',
+      commentsList: [],
+    };
+  }
+  //하트채우기
+  heartLiked = () => {
+    if (this.state.displaying === 'inline-block') {
+      this.setState({ displaying: 'none' });
+      this.setState({ noDisplaying: 'inline-block' });
+    } else {
+      this.setState({ displaying: 'inline-block' });
+      this.setState({ noDisplaying: 'none' });
+    }
+  };
+  //댓글
+  handleInput = e => {
+    this.setState({
+      comment: e.target.value,
+    });
+  };
+  addComments = () => {
+    console.log('good');
+    this.setState({
+      commentsList: this.state.commentsList.concat(this.state.comment),
+      comment: '',
+    });
+  };
+  render() {
+    return (
+      <body>
+        <div className="main">
+          <div className="contentBox">
+            {/* 메뉴바  */}
+            <Nav />
+            {/* 메인컨텐츠_ */}
+            <div className="mainContent">
+              <article className="article">
+                <div className="content_1">
+                  <img
+                    className="con1_img"
+                    alt="프로필사진"
+                    src="/images/kiwanlee/내얼굴.png"
+                  />
+                  <div className="con1_id">guy2895</div>
+                </div>
+                <div className="content_2">
+                  <img
+                    alt="게시사진"
+                    src="/images/kiwanlee/나이키.PNG"
+                    className="con2_img1"
+                  />
+                  <div className="con2_icon">
+                    <img
+                      className="con2_icon1"
+                      alt="하트"
+                      src="/images/kiwanlee/하트.png"
+                    />
+                    <img
+                      className="con2_icon5"
+                      alt="좋아요된하트"
+                      src="/images/kiwanlee/좋아요하트.png"
+                    />
+                    <img
+                      className="con2_icon2"
+                      alt="댓글"
+                      src="/images/kiwanlee/댓글.png"
+                    />
+                    <img
+                      className="con2_icon3"
+                      alt="공유"
+                      src="/images/kiwanlee/공유.png"
+                    />
+                    <img
+                      className="con2_icon4"
+                      alt="즐찾"
+                      src="/images/kiwanlee/즐찾.png"
+                    />
+                  </div>
```

Comment:
```suggestion
                  <div className="con2_icon">
                    {
                      IMAGES.map((ele, idx) => {
                        return (
                            <img
                                key={idx}
                                className={ele.name}
                                alt={ele.alt}
                                src={ele.img}
                             />
                        )
                      })
                    }
                  </div>
```
이런식으로 간단하게 바꿔줄 수 있습니다!!
단 map의 주체인 IMAGES 라는 배열을 만들어서 사용해야겠죠?
힌트를 드리자면, 아래와 같이 만들어서 사용하시면 됩니다.

```
 const IMAGES = [
  {name: con2_icon1, alt="하트", img: "/images/kiwanlee/하트.png"}
];
```